docs/getting-started-tutorial/6-adding-usernames.adoc

```diff
@@ -0,0 +1,153 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# Adding Usernames
+
+We're about to add another use case to our chat system:
+
+ - As a chatter, I want to see who posted a message so that I have better
+   context for the conversation.
+
+The first thing we'll do is change the HTML to look like we want it to. Let's
+add the username:
+
+```
+...
+      <li>
+        <span class="poster">Antonio</span> <span class="body">Hi!</span>
+      </li>
+      <li class="clearable">
+        <span class="poster">David</span> <span class="body">Oh, hey there.</span>
+      </li>
+      <li class="clearable">
+        <span class="poster">Antonio</span> <span class="body">How are you?</span>
+      </li>
+      <li class="clearable">
+        <span class="poster">Antonio</span> <span class="body">Good, you?</span>
```

Comment:
You talking to yourself there? 😛

Reply:
![See, here I am now by myself, talking to myself. That's Chaos Theory](https://m.popkey.co/46a446/Y0dOV.gif)